### ISSUE TITLE
Sorting operations

### ISSUE THREAD

**Issue by mikera** [mikera/core.matrix#83]
Placeholder issue for ordering / sorting operations.

Key requirements:
- Sort arrays based on values
- Sort along any dimension
- Ascending / descending sorts?

**Comment by sethyuan** [comment]
Just curious, is there any progress on this?

**Comment by mikera** [comment]
Yes we have some functionality now in this area... still a bit experimental but check out `select` and related functions that enable arbitrary sorting / re-ordering.

**Comment by sethyuan** [comment]
Cool! will definitely check it out. :)

Best regards

-------- 原始邮件 --------
发件人：Mike Anderson notifications@github.com
时间：周五 8月15日 09:52
收件人："mikera/core.matrix" core.matrix@noreply.github.com
抄送：Seth Yuan sethyuan@gmail.com
主题：Re: [core.matrix] Sorting operations (#83)
